presto-main/src/main/java/com/facebook/presto/operator/TableFinishOperator.java

```diff
@@ -141,11 +183,17 @@ public void addInput(Page page)
     @Override
     public Page getOutput()
     {
+        if (!isBlocked().isDone()) {
+            return null;
+        }
+
         if (state != State.FINISHING) {
             return null;
         }
         state = State.FINISHED;
 
+        List<ComputedStatistics> statistics = getComputedStatistics();
+        System.out.println(statistics);
```

Comment:
leftover debugging line?  But it doesn't look like `statistics` gets used anywhere else.  I'm confused about how this works.

Reply:
It is replaced in a very next commit. I'm going to remove it actually.